src/leagues/heroes/heroes.service.ts

```
@@ -31,15 +32,13 @@ export class LeaguesHeroesService {
       await queryRunner.manager.save(hero);
 
       await queryRunner.commitTransaction();
-    } catch (error) {
+    } catch {
       await queryRunner.rollbackTransaction();
```

why you ignore the error ?